### PR TITLE
feat(admin landing page): created landing page for admin

### PR DESCRIPTION
**Pull request by roymaniv**
Issues 1200, 1201, 1202, 1203

### CODE REVIEW

File: client/src/components/App/Content/Content.tsx [hunk 1]
```diff
@@ -1,25 +1,35 @@
...
     return (
         <>
             <Switch>
                 <Route path={investigationFormRoute} component={InvestigationForm} />
                 <Route path={landingPageRoute} component={LandingPage} />
+                <Route path={adminLandingPageRoute} component={adminLandingPage} />
                 <Route path={usersManagementRoute} component={UsersManagement} />
-                <Redirect from={indexRoute} to={landingPageRoute}/>
+                {
+                    (userType === UserType.ADMIN || userType === UserType.SUPER_ADMIN) ?
```
Comment (EmilyLip) [inline client/src/components/App/Content/Content.tsx:27]: Once the landing page opens and once the table, this is probably an asynchronous problem on the userType

File: client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/InvestigationPieChart/InvestigationPieChart.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { ResponsivePie } from '@nivo/pie';
+
+import InvestigationChartData from './InvestigationChartData';
+
+const InvestigationPieChart: React.FC = (): JSX.Element => {
+    return (
+        <ResponsivePie
+            data={InvestigationChartData}
```
Comment (EmilyLip) [inline client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/InvestigationPieChart/InvestigationPieChart.tsx:9]: add font to the labels 
image

File: client/src/components/App/Content/Content.tsx [hunk 3]
```diff
@@ -1,25 +1,38 @@
...
                 <Route path={usersManagementRoute} component={UsersManagement} />
-                <Redirect from={indexRoute} to={landingPageRoute}/>
+                {
+                    (userType === UserType.INVESTIGATOR) &&
+                    <Redirect from={indexRoute} to={landingPageRoute} />
+                }
+                {
+                    (userType === UserType.ADMIN || userType === UserType.SUPER_ADMIN) &&
+                    <Redirect from={indexRoute} to={adminLandingPageRoute} />
+                }
```
Comment (MayaSasson) [inline client/src/components/App/Content/Content.tsx:33]: do it as if...else and not two ifs

File: client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCard.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+                    onChange={() => { }}
+                />
+            </CardContent>
+            <CardActions style={{ direction: 'ltr', paddingLeft: '1vw' }}>
+                <Button
+                    className={classes.updateButton}
+                    variant='contained'
+                    size='small'>
+                    עדכון
+                    </Button>
```
Comment (MayaSasson) [inline client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCard.tsx:28]: indant

File: client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCard.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+            <CardContent className={classes.TimeRangeCardContent}>
+                <Typography variant='h6' className={classes.cardTitle}>
+                    <b>טווח זמנים</b>
+                </Typography>
+                <DatePick
+                    value={new Date()}
+                    onChange={() => { }}
+                />
+            </CardContent>
+            <CardActions style={{ direction: 'ltr', paddingLeft: '1vw' }}>
```
Comment (MayaSasson) [inline client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCard.tsx:22]: why style?

File: client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCardStyles.ts [hunk 6]
```diff
@@ -0,0 +1,23 @@
+import { Theme } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    timeRangeCard: {
+        width: '15vw',
+        height: '105%',
```
Comment (MayaSasson) [inline client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCardStyles.ts:7]: why precantage?

File: client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCardStyles.ts [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+        borderRadius: '1vw',
+    },
+    cardTitle: {
+        fontSize: '0.9vw',
```
Comment (MayaSasson) [inline client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCardStyles.ts:11]: You have both variant prop and font size, which means one of them is unnecssary... I think you should delete the class

File: client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCardStyles.ts [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+    },
+    TimeRangeCardContent: {
```
Comment (MayaSasson) [inline client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCardStyles.ts:13]: lower case

File: client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCardStyles.ts [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+        height: '52%',
```
Comment (MayaSasson) [inline client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCardStyles.ts:14]: why precentage?

File: client/src/components/App/Content/LandingPage/adminLandingPage/desksFilterCard/desksFilterCard.tsx [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+                }
+            </CardContent>
+            <CardActions style={{ direction: 'ltr', paddingLeft: '1vw' }}>
+                <Button
+                    className={classes.updateButton}
+                    variant='contained'
+                    size='small'>
+                    עדכון
+                </Button>
+            </CardActions>
```
Comment (MayaSasson) [inline client/src/components/App/Content/LandingPage/adminLandingPage/desksFilterCard/desksFilterCard.tsx:38]: Maybe you should export the button to be common if used on both filters?

File: client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/investigationsInfo.tsx [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+    const classes = useStyles();
+
+    let history = useHistory();
+
+    return (
+        <Card className={classes.filtersCard}>
+            <CardActions>
+                <Grid container>
+                    <Grid item xs={3}>
+                        <div style={{ height: '14vh' }}>
```
Comment (MayaSasson) [inline client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/investigationsInfo.tsx:24]: why style?

File: client/src/components/App/Content/LandingPage/adminLandingPage/desksFilterCard/desksFilterCard.tsx [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+import { Button, Card, CardActions, CardContent, Typography } from '@material-ui/core';
+
+import CustomCheckbox from 'commons/CheckBox/CustomCheckbox';
+
+import useStyles from './desksFilterCardStyles';
+
+const DesksFilterCard: React.FC = (): JSX.Element => {
+    const classes = useStyles();
+
+    const allDesks = ['דסק א', 'דסק ב', 'דסק ג'];
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/adminLandingPage/desksFilterCard/desksFilterCard.tsx:11]: why hardcoded?

File: client/src/components/App/Content/Content.tsx (hunk 3 above)
Comment (roymaniv) [reply client/src/components/App/Content/Content.tsx:33]: it was previously if else but it created async problems so I separated it to two conditions to fix it

File: client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCardStyles.ts (hunk 7 above)
Comment (roymaniv) [reply client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCardStyles.ts:11]: I removed the variant

File: client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCard.tsx (hunk 5 above)
Comment (roymaniv) [reply client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCard.tsx:22]: works only with style

File: client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/InvestigationBarChart/InvestigationBarChart.tsx [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+    return (
+        <ResponsiveBar
+            data={InvestigationChartData}
+            borderRadius={9}
+            axisLeft={null}
+            enableGridX={false}
+            enableGridY={false}
+            enableLabel={false}
+            isInteractive={false}
+
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/InvestigationBarChart/InvestigationBarChart.tsx:16]: delete row

File: client/src/components/App/Content/LandingPage/adminLandingPage/desksFilterCard/desksFilterCard.tsx (hunk 10 above)
Comment (roymaniv) [reply client/src/components/App/Content/LandingPage/adminLandingPage/desksFilterCard/desksFilterCard.tsx:34]: fixed

File: client/src/components/App/Content/LandingPage/adminLandingPage/desksFilterCard/desksFilterCard.tsx (hunk 12 above)
Comment (roymaniv) [reply client/src/components/App/Content/LandingPage/adminLandingPage/desksFilterCard/desksFilterCard.tsx:12]: separate user story

**Review by MayaSasson — DISMISSED** [review]

File: client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCard.tsx [hunk 14]
```diff
@@ -0,0 +1,30 @@
...
+            <CardContent className={classes.timeRangeCardContent}>
+                <Typography className={classes.cardTitle}>
+                    <b>טווח זמנים</b>
+                </Typography>
+                <DatePick
+                    value={new Date()}
+                    onChange={() => { }}
+                />
+            </CardContent>
+            <CardActions style={{ direction: 'ltr', paddingLeft: '1vw' }}>
```
Comment (NaamanTsur) [inline client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCard.tsx:23]: Move hard-coded styles to class.

File: client/src/components/App/Content/LandingPage/adminLandingPage/UnassignedCard/UnassignedCard.tsx [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+const UnassignedCard: React.FC = (): JSX.Element => {
+    const classes = useStyles();
+
+    let history = useHistory();
+
+    return (
+        <Tooltip title={unnasignedInvestigationsText}>
+            <Card className={classes.unassignedCard} onClick={() => {}}>
+                <div className={classes.investigationAmount}>
+                    <Typography className={classes.investigationNumberText}><b>13</b></Typography>
```
Comment (NaamanTsur) [inline client/src/components/App/Content/LandingPage/adminLandingPage/UnassignedCard/UnassignedCard.tsx:19]: Should the data be hard-coded for now?

File: client/src/components/App/Content/LandingPage/adminLandingPage/desksFilterCard/desksFilterCard.tsx [hunk 16]
```diff
@@ -0,0 +1,39 @@
...
+                            checkboxElements={[{
+                                key: desk,
+                                value: desk,
+                                labelText: desk,
+                            }]}
+                        />
+                    ))
+                }
+            </CardContent>
+            <CardActions style={{ direction: 'ltr', paddingLeft: '1vw' }}>
```
Comment (NaamanTsur) [inline client/src/components/App/Content/LandingPage/adminLandingPage/desksFilterCard/desksFilterCard.tsx:32]: Hard-coded styles should be moved to class

**Review by NaamanTsur — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/LandingPage/adminLandingPage/UnassignedCard/UnassignedCard.tsx (hunk 15 above)
Comment (RoeiRom) [reply client/src/components/App/Content/LandingPage/adminLandingPage/UnassignedCard/UnassignedCard.tsx:19]: yes in the future there will be integration with DB

File: client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/investigationsInfo.tsx [hunk 17]
```diff
@@ -0,0 +1,50 @@
...
+    const classes = useStyles();
+
+    let history = useHistory();
+
+    return (
+        <Card className={classes.filtersCard}>
+            <CardContent>
+                <Grid container>
+                    <Grid item xs={12} className={classes.investigationInfoButtonWrapper}>
+                        <div style={{ height: '14vh', width: '11vw' }}>
```
Comment (NaamanTsur) [inline client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/investigationsInfo.tsx:24]: Sorry missed this: more hard-coded styles to classes

**Review by NaamanTsur — CHANGES_REQUESTED** [review]

**Review by NaamanTsur — APPROVED** [review]

**Review by MayaSasson — APPROVED** [review]

**Pull request merged**